contracts/handler/SwapHandlers/PancakeSwapHandler.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.17;
+
+import {ISwapHandler} from "../ISwapHandler.sol";
+
+contract PancakeSwapHandler is ISwapHandler {
```

Comment:
This contract has the same implementation as the UniswapHandler.. you can initialize the router address so we can reuse the handler for uniswap forks